docs/FAQ.md

```diff
@@ -44,3 +44,16 @@ $ export QUILT_MINIMIZE_STDOUT=true
 1. Go to CloudFormation > Stacks > YourQuiltStack > Outputs
 1. Copy the row labeled TemplateBuildMetadata
 1. "git_revision" is your template version
+
+## Hashing takes a long time. Can I speed it up?
+
+Yes. Follow these steps:
+
+1. Run your compute in the same region as your S3 bucket (as opposed to
+a local machine or foreign region)—I/O is much faster.
```

Comment:
```suggestion
a local machine or foreign region) — I/O is much faster.
```

Reply:
Correct as written and would rather keep style consistent. It looks much better in a variable width font.